Gemfile

```diff
@@ -40,13 +41,15 @@ group :development, :test do
   gem 'rubocop', '~> 1.12'
   gem 'rubocop-rails', '~> 2.9'
   gem 'rubocop-rspec'
-  gem 'simplecov'
-  gem 'simplecov-json'
 end
 
 group :test do
   gem 'database_cleaner'
   gem 'shoulda-matchers'
+  gem 'webmock'
+  gem 'simplecov'
+  gem 'simplecov-json'
+  gem 'vcr'
 end
```

Comment:
added webmock and VCR gem. Together you can make real HTTP requests in your tests and then store them to "play back" (so you can test real request/response flow without actually making the requests)